utils/calendrical_calculations/src/persian.rs

```diff
@@ -73,6 +112,27 @@ fn arithmetic_persian_year_from_fixed(date: RataDie) -> i64 {
     }
 }
 
+/// arithmetic_persian_year_from_fixed modified for the 33-year rule
+fn fast_persian_year_from_fixed(date: RataDie) -> i64 {
+    let days_since_epoch = date - fixed_from_fast_persian(1, 1, 1);
```

Comment:
Question: is `fixed_from_fast_persian(1, 1, 1)` the same as `FIXED_PERSIAN_EPOCH`? If so, maybe use the constant to reduce the number of function calls.

Reply:
It's one day away. Replaced.